gbpservice/nfp/orchestrator/config_drivers/heat_driver.py

```diff
@@ -644,10 +644,11 @@ def _update_node_config(self, auth_token, tenant_id, service_profile,
         base_mode_support = (True if service_details['device_type'] == 'None'
                              else False)
 
-        stack_template = service_chain_node.get('config')
-        stack_template = (jsonutils.loads(stack_template) if
-                          stack_template.startswith('{') else
-                          yaml.load(stack_template))
+        _, stack_template_str = self.parse_template_config_string(
```

Comment:
Can tag_str be any of the other values such as 'config_init' and 'ansible' here?